tachyon/crypto/commitments/vector_commitment_scheme.h

```diff
@@ -94,8 +94,10 @@ class VectorCommitmentScheme {
 
   // Verify an opening |proof| that proves that |members| belong to a
   // |commitment|.
+  // NOTE(chokobole): const was removed from |Commitment| since it can be a
+  // |Transcript|. At this moment, |WriteToTranscript()| is not a const method.
```

Comment:
Is there any case where `WriteToTranscript` is needed in verification?
Plus, wouldn't `proof` be more appropriate for variables for getting transcripts?

Reply:
1. `ReadFromProof` internally calls `WriteToTranscript`! 
2. I don't understand a question, I'll ask this at office tomorrow!

Reply:
`Commitment` sometimes can be a `Transcript`.